data/coalition/heliarch intro.txt

```diff
@@ -0,0 +1,1831 @@
+			`	"Not fond of outsiders, the people of this world are," Tugroob explains. "Decided to employ undercover Kimek agents, the consul has."`
+			`	You wait around for a couple hours as Iplora and Tugroob tinker with your ship's communication devices, keeping in contact with the two groups. They don't mention if either group finds anything, and as night falls, Kirrie returns to the ship on his own.`
+
+
+
+mission "Heliarch Recon 1"
+	name "Heliarch Reconnaissance"
+	description "Fly to all the Quarg planets and stations the Heliarch asked, land on them, letting your modified ship sensors scan their surroundings, then head back to the <destination>."
+	minor
+	source
+		government "Heliarch"
+	stopover "Lagrange"
+	stopover "Forpelog"
+	stopover "Grakhord"
+	stopover "Alta Hai"
+	stopover "Kuwaru Efreti"
+	destination "Ring of Wisdom"
+	to offer
+		"reputation: Quarg" >= 0
+		"coalition jobs" >= 55
+		has "Wanderers: Quarg Assistance 2: done"
+		has "Coalition: Contributor: done"
+		not "Resistance: Questions: done"
+	on offer
+		require "Jump Drive"
+		conversation
+			`Three Heliarch agents are waiting for you when you land. "The human visitor who so much to our society has contributed, we greet," the Saryd says as they all salute you.`
+			`	"Sent by Consul Aulori, we were," the Kimek says proudly, as if you were supposed to know who that is. "An important task for you, he has in mind."`
+			`	"How far would the visitor go, if to further help our Coalition, it meant?" The Arach asks.`
+			choice
+				`	"I'm not sure I understand what you're getting at."`
+				`	"Obviously I have my limits as for what I would do."`
+			`	"A jump drive, you possess. Far away, you can travel. Bother you, the harsh cages of the Quarg do not," the Saryd explains as all three walk toward your ship.`
+			`	"Few in number, our own jump drives are, and too risky would it be, for one of our own to venture far out," the Kimek adds.`
+			`	"If to help us gain knowledge of the opressors, you wish, lend the Coalition your ship's sensors for a short time, would you?" Asks the Arach.`
+			choice
+				`	"Are you asking me to spy on the Quarg?"`
+				`	"Sorry, I don't want to get involved in your conflict with the Quarg."`
+					decline
+			`	"As espionage, a 'malfunction' qualifies not," the Saryd responds.`
+			`	"Merely alter your own ship's sensors, we would. For a time, specially tuned to scan Quarg worlds, they would be," the Kimek clarifies. "No evidence against you, they would have."`
+			choice
+				`	"Alright, which worlds do you need me to land on?"`
+				`	"Sorry, I don't want to get involved in your conflict with the Quarg."`
+					decline
+			`	They ask you to show them your map of the galaxy, so you bring them inside your ship, pointing to them all the Quarg planets and stations you have found. They're a bit startled when they first see the entire map, as if unsure about where to send you.`
```

Comment:
The player character presumably knows about the Quarg from videos and such media they consumed prior to their first encounter with one, I imagine it'd also be something they know about that the Quarg (in human space) are near Tarazed.
Also don't think it'd be very likely for someone that has the Wanderers mission done wouldn't have at least gone close to the Quarg pocket there (actually think that's straight up impossible given you need FW completed to get to that portion of the Wanderer campaign, and in FW you have to bring Alondo to Tarazed).
In any case I don't think it's necessary to add more conditions for this to trigger, it's reasonable to assume that the player could at least point out the area the Quarg in human space are, I'd say.

Reply:
I don't mind it, what I mostly meant was you don't have to put that Quarg missions as requirement, since the Heliarchs are enemies of the Quarg, why would you need to do missions with them, you could just have "visited planet" for the two ringworlds.

Reply:
Oh, I just thought it was easier to have that mission as the requirement, but yeah it's a bit restrictive to have this chain locked behind that far into Wanderers.
I'll look into changing the conditions to checking for visits on Alta Hai and Kuwaru Efreti.